hugo/layouts/partials/head.html

```diff
@@ -25,8 +25,19 @@
     <!-- Establishing early connections to speed up downloads -->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="preconnect" href="https://code.jquery.com">
     <link rel="preconnect" href="https://cdn.jsdelivr.net">
```

Comment:
I wonder: why `preconnect` instead of `preload` ? I read in the docs you linked to: "In general, try to use <link rel="preload">, as it's a more comprehensive performance tweak" 🤔

Reply:
Thanks, bit weird that lighthouse makes the warning about preconnect but the recommends preload. My new understanding now is that I should probably use preload for everything except things that are non-blocking. I've made a Jira task (EB-197) for this as well as I think it could take some experimentation.